test/integration/http2_flood_integration_test.cc

```diff
@@ -1254,6 +1261,9 @@ TEST_P(Http2FloodMitigationTest, UpstreamZerolenHeader) {
 
 // Verify that the HTTP/2 connection is terminated upon receiving invalid HEADERS frame.
 TEST_P(Http2FloodMitigationTest, UpstreamZerolenHeaderAllowed) {
+#ifdef ENVOY_ENABLE_UHV
```

Comment:
maybe make this a utility function in the base test?

Reply:
Done